config/initializers/cancan_hacks.rb

```diff
@@ -1 +1,54 @@
-require 'cancan_hacks'
+# This makes CanCan work with Inherited resources.
+#
+# Inspiration comes from hexors's comment over here:
+# http://github.com/ryanb/cancan/issues#issue/23
+#
+# TODO: Maybe this is no longer needed, as there is a new version of canca. Check it.
```

Comment:
What happened with this @duduribeiro ?

Reply:
Instead of requiring it from lib, I moved the code from `lib/cancan_hacks` to here

Reply:
@duduribeiro I mean this comment, the "Maybe this is not needed anymore because this is for a very old version of cancan"

Reply:
Like... did you investigate if we still need this? because if we don't need it, let's remove it 😄

Reply:
ah, gotcha!.. just grabbed the code from there. I think this is fixed by https://github.com/ryanb/cancan/commit/4eee637270e524f3a56341bd595559a96a996551. Will try to update the gem to check if it works!

thanks

Reply:
We spoke in private, conclusion here:
Cadu thinks that is fixed since 1.6.10, and we have 1.7 in our Gemfile, so it seems safe to be removed 😎